build.rs

```diff
@@ -1,7 +1,119 @@
 use std::env;
+use std::ffi::CStr;
+use std::os::raw::c_char;
+
+#[derive(PartialEq, Eq, PartialOrd, Ord)]
+struct Version {
+    major: usize,
+    minor: usize,
+    micro: usize,
+}
+
+impl Version {
+    fn new(major: usize, minor: usize, micro: usize) -> Version {
+        Version {
+            major,
+            minor,
+            micro,
+        }
+    }
+
+    fn parse(s: &str) -> Result<Version, Box<dyn std::error::Error>> {
+        let err = format!("invalid pcap lib version: {}", s);
+
+        let re = regex::Regex::new(r"([[:digit:]]+)\.([[:digit:]]+)\.([[:digit:]]+)")?;
+        let captures = re.captures(s).ok_or(err.clone())?;
+
+        let major_str = captures.get(1).ok_or(err.clone())?.as_str();
+        let minor_str = captures.get(2).ok_or(err.clone())?.as_str();
+        let micro_str = captures.get(3).ok_or(err.clone())?.as_str();
+
+        Ok(Version::new(
+            major_str.parse::<usize>()?,
+            minor_str.parse::<usize>()?,
+            micro_str.parse::<usize>()?,
+        ))
+    }
+}
+
+fn get_pcap_lib_version() -> Result<Version, Box<dyn std::error::Error>> {
+    if let Ok(libver) = env::var("PCAP_VER") {
```

Comment:
Change `PCAP_VER` to `LIBPCAP_VER` to avoid mix up with the crate version?

Reply:
Done. I also changed `PCAP_LIBDIR` to `LIBPCAP_LIBDIR` for consistency (added to changelog as well)